### ISSUE TITLE
Some visual glitches

### ISSUE THREAD

**Issue by hgraeber** [FreeSpacenav/spnavcfg#19]
There some small visual glitches in the dialog:

- In the Translation line for TX and the Rotation line for RX the text in the device axis drop down is more left then in the lines for TY, TZ, RY and RZ. Maybe centering would be better here.
-  The fields with spin boxes for Translation and Rotation have no borders, except the ones in the RY lines. This shall be consistent. Maybe having borders in all the fields would be better.

![spnavcfg-1 0](https://user-images.githubusercontent.com/4057854/165292613-ea54877e-50d7-4d24-8df7-4ba18910cb85.png)

**Comment by jtsiomb** [comment]
It doesn't look like that here, so I can't be sure if I actually changed anything, but I just commited a change which *might* fix the first one of your issues. I looked through the designer-generated UI definition file, and indeed there was a slight difference in the definition of the TX and RX combo boxes, so I manually edited it to match the rest.

About your second issue, I could not detect any difference between how `spin_sens_ry` and every other `spin_sens_*` is defined, so I can't see why it should be rendered differently by qt.

Please test the latest commit and let me know if the first issue is fixed or not.

**Comment by hgraeber** [comment]
I am sorry that my bug report is a little bit inaccurate. I forget to mention that I used a high dpi screen. It' is possible that this does matter.

I will test your changes and have a look into the code, knowing C++ and Qt myself. Maybe I even can make provide pull request.

**Comment by hgraeber** [comment]
I have play a little bit around with different monitors and different dpi. That makes no difference in behavior. In one case I have got the right appearance. In this case it is possible to enforce the wrong visual appearance by changing the dialogs size with the mouse. 

Looking close onto the dialog, one can see, that the rows are not properly aligned. Reason is that there are not 4 rows in the Translation and Rotation group boxes, but 6 with some unneeded rowspans. Removing the rowspans fixes the alignment on all machines I have available for tests. But it is still possible to enforce it by making the dialog vertically smaller using the mouse. What changes is the the only row affected is the TY row.

**Comment by jtsiomb** [comment]
I'm sorry I can't seem to reproduce at all. I'm resizing it all over the place and never get misaligned widgets.

Could it be a Qt bug? are you by any chance using the wayland backend of Qt, which I guess is not as well-tested as the standard X11 one?

**Comment by hgraeber** [comment]
Unfortunately there are many components which may influence this behavior:
- Wayland / X11
- High DPI / scaling factor
- Qt style
- Qt version
I have tested many combinations and found out, that it's the style is the culprit. I use KDEs Breeze and there is a bug report for it: [https://bugs.kde.org/show_bug.cgi?id=416655](url). So I close this issue.

Nevertheless, I have made some other changes which I will provide as a PR.
